.ci/pitest-survival-check-xml.groovy

```diff
@@ -102,12 +93,37 @@ private static int checkPitestReport(String profile, String flag) {
     final Set<Mutation> survivingMutations = getSurvivingMutations(mutationReportNode)
 
     final File suppressionFile = new File(suppressedMutationFileUri)
-    Set<Mutation> suppressedMutations = Collections.emptySet()
+    Set<Mutation> suppressedMutations = new TreeSet<>()
     if (suppressionFile.exists()) {
         final Node suppressedMutationNode = xmlParser.parse(suppressedMutationFileUri)
         suppressedMutations = getSuppressedMutations(suppressedMutationNode)
```

Comment:
Correct me if I am wrong, but this doesn't handle `isUnstable`? If we set something as unstable, will it keep it as unstable when writting out the new suppression file or will it incorrectly flip it's status from true to false? Also, if something is unstable and it is the only thing there, won't it constantly say the file needs to be deleted and recreated?

If so, lets discuss this in another issue/PR.

Reply:
No, the autofix capability does not handle unstable mutations, in fact it will change unstable mutations to stable. Let's keep this for a separate PR. I think this is a bug in checker-framework script too.

Reply:
@Vyom-Yadav Please create an issue for this.